aten/src/ATen/native/mkldnn/RegisterMkldnnOpContextClass.cpp

```diff
@@ -42,6 +44,9 @@ TORCH_LIBRARY(mkldnn_prepacked, m) {
 
   m.def(TORCH_SELECTIVE_SCHEMA(
       "mkldnn_prepacked::conv2d_run(Tensor X, __torch__.torch.classes.mkldnn.ConvOpContext W_prepack) -> Tensor Y"));
+
+  m.def(TORCH_SELECTIVE_SCHEMA(
+    "mkldnn_prepacked::linear_eltwise(Tensor X, Tensor W, Tensor? B, str attr, Scalar?[] scalars, str? algorithm) -> Tensor Y"));
```

Comment:
Is this schema general enough to cover all the elementwise operations regardless of its parameter number? Seems like yes as the `scalars` is a vector, but just confirm with you.

Reply:
Yes, this schema covers all the targeted elementwise operators.

Reply:
it is confused with using mkldnn_prepacked for this op, it does not use prepack weight. also same as LinearPreapck file name.

Reply:
The OP is renamed to `torch.ops.mkldnn._linear_pointwise`.